app/src/main/java/org/jellyfin/androidtv/data/querying/AdditionalPartsQuery.java

```diff
@@ -0,0 +1,13 @@
+package org.jellyfin.androidtv.data.querying;
```

Comment:
All new files should be written in Kotlin

Reply:
Done.